safekeeper/src/safekeeper.rs

```diff
@@ -30,8 +30,8 @@ use utils::{
 };
 
 pub const SK_MAGIC: u32 = 0xcafeceefu32;
-pub const SK_FORMAT_VERSION: u32 = 4;
-const SK_PROTOCOL_VERSION: u32 = 1;
+pub const SK_FORMAT_VERSION: u32 = 5;
+const SK_PROTOCOL_VERSION: u32 = 2;
```

Comment:
We're updating protocol here, so I'd also added current safekeeper ID (`ZNodeId`) to `AcceptorGreeting`.

Reply:
Makes sense, added.